account_einvoice_generate/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018-2020 Akretion France (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account e-invoice Generate",
+    "version": "14.0.1.0.0",
+    "category": "Accounting & Finance",
+    "license": "AGPL-3",
+    "summary": "Technical module to generate PDF invoices with " "embedded XML file",
+    "author": "Akretion,Onestein,Odoo Community Association (OCA)",
+    "maintainers": ["alexis-via"],
+    "website": "https://github.com/OCA/edi",
+    "depends": ["account"],
+    "excludes": ["account_edi"],
```

Comment:
Why do you need this exclusion?

Reply:
I want to make things clear that you have to make a choice: either you opt for the einvoicing stack of OCA, or for the einvoicing stack of Odoo SA. You can't use both at the same time.
From a technical point of view, the exclusion is one level higher : account_invoice_factux excludes account_edi_facturx. But I prefer to make things clear at the generic level, so that there is no misunderstanding.